build.gradle

```diff
@@ -304,13 +311,21 @@ task windowsPatches(type:Exec) {
 
 task cmakeJniLib(type:Exec) {
     workingDir 'jni'
+    def args = []
+    args.add("cmake")
+    args.add(".")
+    args.add("-DKNN_PLUGIN_VERSION=${opensearch_version}")
+    args.add("-DSIMD_ENABLED=${simd_enabled}")
+    args.add("-DCOMMIT_LIB_PATCHES=${commit_lib_patches}")
+    args.add("-DAPPLY_LIB_PATCHES=${apply_lib_patches}")
     if (Os.isFamily(Os.FAMILY_WINDOWS)) {
         dependsOn windowsPatches
-        commandLine 'cmake', '.', "-G", "Unix Makefiles", "-DKNN_PLUGIN_VERSION=${opensearch_version}", "-DBLAS_LIBRARIES=$rootDir\\src\\main\\resources\\windowsDependencies\\libopenblas.dll", "-DLAPACK_LIBRARIES=$rootDir\\src\\main\\resources\\windowsDependencies\\libopenblas.dll", "-DSIMD_ENABLED=${simd_enabled}", "-DCOMMIT_LIB_PATCHES=${commit_lib_patches}"
-    }
-    else {
-        commandLine 'cmake', '.', "-DKNN_PLUGIN_VERSION=${opensearch_version}", "-DSIMD_ENABLED=${simd_enabled}", "-DCOMMIT_LIB_PATCHES=${commit_lib_patches}"
```

Comment:
did we skip `-DKNN_PLUGIN_VERSION=${opensearch_version}` intentionally

Reply:
See line# 317.